src/shared-components/accordion/style.ts

```diff
@@ -98,30 +98,34 @@ export const Container = styled.div<{
   background-color: ${({ theme }) => theme.COLORS.white};
   max-width: ${BREAKPOINTS.md}px;
 
-  ${({ borderRadius = '4px' }) => `
+  ${({ borderRadius, theme }) => {
+    const borderRadiusValue = borderRadius || theme.BORDER_RADIUS.small;
```

Comment:
Since this PR is concerned with implementing the style guide prescription for how `border-radius` should operate, I think we should define an enum for what the acceptable `border-radius` values are, in order to actually enforce it. 

It could get thorny because most user-land usage is probably in `rem` but we're using mostly `pixels` in this PR, but something like this might work:

`type BorderRadiusValues = valueof<ThemeType['BORDER_RADIUS']> | '0.25rem' | '0.5rem' | '2rem'`